content/training/R_Basics/Advanced_Functions.Rmd

```diff
@@ -107,55 +111,64 @@ fct_count(linelist_cleaned$sex)
 *Note: Use the argument `only = ` to drop specific levels: `fct_drop(linelist_cleaned$sex, only = "U")`*
 
 
-### The %in% operator  
+### The `%in%` operator  
 
-The operator %in% looks to see if a value is present within a vector. A vector can be created with the function c(), and a variable in a data frame is also a vector. 
+The `%in%` operator looks to see if any elements of a vector are within another vector. **You can use this to filter your data to a specific set of values.** It's like a parallel version of `==`. A vector can be created with the function `c()`, and a variable in a data frame is also a vector. 
 
-In the simple example below, R is asked if the number 4 is present in the vector "numbers":
+Here, we show how `%in%` can be used to find which random sample of 10 letters are in the set of (a, b, c):
 
 ```{r}
-# Vector is defined
-numbers <- c(4, 10, 5, 365, 12, 0.5, 9)
+# randomly sample 10 letters from the alphabet
```

Comment:
Imagining myself as a relative beginner, starting with set.seed(99) looks scary and makes me confused as to why there's a 99 when we are dealing with letters. I know random selection is a standard, but I think for these purposes it is more simple and less confusing to just define the vector as static. 

Also below I slightly adjusted the wording and example in a way I think makes it easier to interpret. Take what you wish... 

```suggestion
# my_letters is a vector of 10 letters from the alphabet
my_letters <- c("p" "a" "l" "v" "w" "u" "c" "j" "b" "f") 
my_letters

# use %in% to ask "for each letter in my_letters, is it in the list of vowels (a, e, i, o, u)?" 
my_letters_evaluated <- my_letters %in% c("a", "e", "i", "o", "u") 

# the result is a logical (TRUE/FALSE) vector
my_letters_evaluated

# this new vector can be used to subset our original vector
# the following returns only the letters in my_letters that evaluated to TRUE in my_letters_evaluated (the vowels).
my_letters[my_letters_evaluated]
```

Reply:
Good point! In that case, I'll make the letters mean something instead of random gobbeldygook